source/_integrations/amcrest.markdown

```diff
@@ -153,7 +153,8 @@ Available services:
 `enable_motion_recording`, `disable_motion_recording`,
 `enable_recording`, `disable_recording`,
 `goto_preset`, `set_color_bw`,
-`start_tour` and `stop_tour`
+`start_tour` and `stop_tour`,
+`ptz_control`
```

Comment:
Shouldn't "and" be between `stop_tour` & `ptz_control`?

Reply:
yep